src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5366,6 +5366,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_NotNullableOperatorNotReferenceType" xml:space="preserve">
     <value>The ! operator can only be applied to reference types.</value>
   </data>
+  <data name="WRN_MissingNonNullTypesContext" xml:space="preserve">
+    <value>The suppression operator should be used in code with a `[NonNullTypes(true/false)]` context.</value>
```

Comment:
>The suppression operator [](start = 11, length = 24)

Perhaps `! operator`, to match with error above and to allow changes to the behavior (if `!` does not suppress warnings for instance).

Reply:
Maybe we change both to be `The suppression operator (!)...`? It would help standardize the way people talk about it.

---
In reply to: [205837209](https://github.com/dotnet/roslyn/pull/28825#discussion_r205837209) [](ancestors = 205837209)